olive/workflows/run/config.py

```diff
@@ -66,7 +66,7 @@ class RunConfig(ConfigBase):
     evaluators: Dict[str, OliveEvaluatorConfig] = None
     pass_flows: List[List[str]] = None
     engine: RunEngineConfig
-    passes: Dict[str, RunPassConfig]
+    passes: Dict[str, RunPassConfig] = None
```

Comment:
just curious if the empty dict {} works for original code without your changes. 
This is not related to your PR

Reply:
New `set_pass_flows` still needs a pass to be registered. But empty dict works after the change for `set_pass_flows`

Reply:
Oh, xiaoyu is right. In my previous changes, I missed this part.
![image](https://github.com/microsoft/Olive/assets/13343117/aacc8cb3-2902-4689-ba64-2fb5044d8881)